bothan-okx/src/worker/error.rs

```diff
@@ -0,0 +1,16 @@
+use crate::api;
+use bothan_core::store;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub(crate) enum WorkerError {
+    #[error("value is not a valid decimal: {0}")]
+    Underflow(#[from] rust_decimal::Error),
```

Comment:
Underflow is just one of the decimal error? should it be InvalidDecimal instead?

Reply:
fixed